export/planloader/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
```

Comment:
Is this needed if you already have this in the root `CMakeLists.txt`?

Reply:
I had left it in as a self documenting reminder (this library requires PIC) but in truth it doesn't need it -- but its dependent libraries do.  Removed.